Naive greedy alg/scheduling/Swarm.py

```diff
@@ -23,14 +24,19 @@ def __init__(self, seed_num, num_particles, num_srvs, num_exp=10):
         self.best_cost = inf
         self.experiment = Experiments()
 
-    def run_epochs(self, num_epochs, draw_stats=False):
+    def run_epochs(self, num_epochs, draw_stats=False, draw_graph=False):
+        epochs_stats = []
         for i in range(num_epochs):
             print("epoch: ", i)
```

Comment:
Remove print

Reply:
I have found the prints pretty useful to monitor how quickly the particles converge and to then decide on the number of epochs to use. I will comment them, if you think that they must be removed, then you can remove them.

Reply:
You can use the Logger instead of the prints, and change its level to enable/disable the prints